test/conformance/testing/include/uur/utils.h

```diff
@@ -170,7 +170,19 @@ inline std::string GetDeviceName(ur_device_handle_t device) {
 }
 
 inline std::string GetPlatformAndDeviceName(ur_device_handle_t device) {
-    return GetPlatformName(GetPlatform()) + "__" + GetDeviceName(device);
+    constexpr int MaxAddressLength = 256;
+    char pciAddress[MaxAddressLength];
+    char id[MaxAddressLength];
+    size_t propSize = sizeof(pciAddress);
```

Comment:
name it rather `pci_adress_len` or actual_len, not `propSize` 😉

Reply:
I've changed it.